code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1441,9 +1445,9 @@
 			M.cure_nearsighted(EYE_DAMAGE)
 			M.blur_eyes(35)
 
-	else if(M.has_trait(TRAIT_NEARSIGHT, EYE_DAMAGE))
+	else if(M.has_trait(TRAIT_NEARSIGHT))
```

Comment:
you aren't supposed to cure nearsighted trait, because you can get buffs, nearsight = +1 point